source/quick-reference.txt

```diff
@@ -0,0 +1,455 @@
+               { year: 2001 },
+               { $inc: { 'imdb.votes': 100 } }
+             );
+
+          .. output::
+             :language: js
+
+             [
+               { title: 'A Beautiful Mind', year: 2001, imdb: { votes: 826257, ... },
+               { title: 'Shaolin Soccer', year: 2001, imdb: { votes: 65442, ... },
+               ...
+             ]
+
+   * - | **Update Arrays in Documents**
+       |
+       | `API Documentation <{+api+}/classes/Collection.html#update>`__
+       | :ref:`Fundamentals <node-fundamentals-update-array>`
+
+     - .. io-code-block::
+          :copyable: true
+
+          .. input::
+             :language: js
+
+             await coll.updateOne(
+               { title: 'Cosmos' },
+               { $push: { genres: 'Educational' } }
+             ):
+
+          .. output::
+             :language: js
+
+             { title: 'Cosmos', genres: [ 'Documentary', 'Educational' ] }
+
+   * - | **Replace a Document**
+       |
+       | `API Documentation <{+api+}/classes/Collection.html#replaceOne>`__
+       | :ref:`Usage Example <node-usage-replaceone>`
+       | :ref:`Fundamentals <node-fundamentals-replaceone>`
+
+     - .. io-code-block::
+          :copyable: true
+
+          .. input::
+             :language: js
+
+             await coll.replaceOne(
+               { name: 'Deli Llama', address: '2 Nassau St' },
+               { name: 'Lord of the Wings', zipcode: 10001 }
+             );
+
+          .. output::
+             :language: js
+
+             { name: 'Lord of the Wings', zipcode: 10001 }
+
+   * - | **Delete a Document**
+       |
+       | `API Documentation <{+api+}/classes/Collection.html#deleteOne>`__
+       | :ref:`Usage Example <node-usage-deleteone>`
+       | :ref:`Fundamentals <node-fundamentals-delete>`
+
+     - .. code-block:: javascript
+          :copyable: true
+
+          await coll.deleteOne({ title: 'Congo' });
+
+   * - | **Delete Multiple Documents**
+       |
+       | `API Documentation <{+api+}/classes/Collection.html#deleteMany>`__
+       | :ref:`Usage Example <node-usage-deletemany>`
+       | :ref:`Fundamentals <node-fundamentals-delete>`
+
+     - .. code-block:: javascript
+          :copyable: true
+
+          await coll.deleteMany({ title: { $regex: /^Shark.*/ } });
+
+   * - | **Bulk Write**
+       |
+       | `API Documentation <{+api+}/classes/Collection.html#bulkWrite>`__
+       | :ref:`Usage Example <node-usage-bulk>`
+
+     - .. io-code-block::
+          :copyable: true
+
+          .. input::
+             :language: js
+
+             await coll.bulkWrite([
+               {
+                 insertOne: {
+                   document: {
+                     title: 'A New Movie',
+                     year: 2022
+                   }
+               },
+                 deleteMany: {
+                   filter: { year: { $lt: 1970 } }
+                 }
+               }
+             ]);
```

Comment:
issue: I may have missed this in my previous review, but the indentation/brackets seem off here?

```suggestion
               {
                 insertOne: {
                   document: {
                     title: 'A New Movie',
                     year: 2022
                   }
                 }
               },
               {
                 deleteMany: {
                   filter: { year: { $lt: 1970 } }
                 }
               }
             ]);
```

Reply:
Good catch